agent/Gopkg.toml

```diff
@@ -19,7 +19,7 @@
 #  name = "github.com/x/y"
 #  version = "2.4.0"
 
-required = ["github.com/golang/mock/mockgen/model"]
+required = ["github.com/golang/mock/mockgen/model", "github.com/aws/aws-sdk-go/service/efs"]
```

Comment:
why is this separate from the regular aws-sdk-go constraint?

Reply:
Yes I'm going to figure out how to resolve this. The reason was that we are only using the efs client in a linux-specific unit test, so I wasn't able to get dep to recognize the dependency from my work laptop.

Reply:
this is resolved now, deleting the entry and running "dep ensure" again seems to have fixed it